templates/init/functions/javascript/index.js

```diff
@@ -1,9 +1,10 @@
-const functions = require("firebase-functions");
+const { onRequest } = require("firebase-functions/v2/https");
```

Comment:
Optional idea: add some commented-out lines showing other imports, like this:

```suggestion
/**
 * Import function triggers from their respective submodules:
 *
 * const { onCall } = require("firebase-functions/v2/https");
 * const { onDocumentWritten } = require("firebase-functions/v2/firestore");
 *
 * See a full list of supported triggers at https://firebase.google.com/docs/functions
 */

const { onRequest } = require("firebase-functions/v2/https");
```